frontend/appflowy_flutter/pubspec.yaml

```diff
@@ -165,6 +166,7 @@ flutter:
     - assets/images/common/
     - assets/images/grid/setting/
     - assets/translations/
+    - assets/test/workspaces/
```

Comment:
If you put the test resource here, won't it be included in the release product?

Reply:
That's correct. Tomorrow, I'll try to see whether I can exclude those assets from the release build.

Reply:
IIUC, files outside the `lib` directory won't be included in the build unless referenced in the `pubspec.yaml`. Hence, there needs to be some mechanism that removes this line from the pubspec.yaml for the release build.

Reply:
Could be an idea to have integration tests in a separate package (integration test package).

Otherwise we can write a script to amend the pubspec yaml, in that case the CI script should output it and await approval before releases, which is not that nice..

Reply:
I think that having the integration tests as a separate package is a nice idea. It's better than writing a script to modify the `pubspec.yaml` I think that'll be confusing for someone that doesn't have context. I've never had to pull out an integration test suite into a separate package, so I'll let you know how that goes @Xazin.

Reply:
So, we're going to separate the integration_test package now?

Reply:
I tried, but the integration tests can't run without a flutter project so it's not a good option anymore. 

Along the same lines, we can create a separate repo/branch that excludes the test assets with a .gitignore file. Another option is to use a build script to remove the files and updates the pubspec.yaml before running flutter build.